packages/asset/contracts/Asset.sol

```diff
@@ -161,45 +126,6 @@ contract Asset is
         _setURI(id, metadataHash);
     }
```

Comment:
I still don't understand why we need this mintspecial here.
Can't we put that in AssetMinter and call regular mint ?

Reply:
We don't this part will be split into another contract but is not part of this PR.